index.html

```diff
@@ -45,23 +46,22 @@ <h2 class="puzzle__number">1.</h2>
                 которой&nbsp;ОН фигурирует?</p>
             <form class="form">
                 <label class="form__label" for="first-puzzle">Ваш ответ:</label>
-                <input class="form__field" type="number" min="1800" max="2022" id="first-puzzle" name="fisrt-puzzle"
+                <input class="form__field" type="number" min="1800" max="2022" id="first-puzzle" name="first-puzzle"
                     placeholder="Например: 1984">
-                <div class="form__button-container">
-                    <button class="form__button form__button_type_submit" type="submit">Ответить</button>
-                    <button class="form__button form__button_type_hint" type="button">Узнать ответ</button>
-                </div>
             </form>
-            <div class="wrong-answer-board">
+            <div class="form__button-container">
```

Comment:
It feels like we should not make button a part of a form; we are not using `<form>` features, we are implementing custom handlers for buttons.